### ISSUE TITLE
Dashboard resource column's column value defaults to 0, causing plan churn

### ISSUE THREAD

**Issue by nickchappell** [splunk-terraform/terraform-provider-signalfx#84]
`terraform plan` output for column sections of dashboards keeps saying it needs to be updated from **1** to **0**, even after a successful `apply` (IDs and names anonymized):

```bash
  # signalfx_dashboard.hosting_backup_system_superseded_run_dashboard will be updated in-place
  ~ resource "signalfx_dashboard" "backup_system_superseded_run_dashboard" {
        charts_resolution = "default"
        dashboard_group   = "EExR-32e23AE"
        id                = "EExS24rdfAE"
        name              = "Superseded Runs"
        time_range        = "-31d"
        url               = "https://app.signalfx.com/#/dashboard/EEasdfxzAcAE"

      ~ column {
            chart_ids = [
                "EExR_5oAYttsser0",
                "EE2erg4mAgAE",
                "EExSGWUAYAA",
                "EExSasdfAYAA",
                "EExzxcvlAYAA",
                "EExSGJVAYAE",
                "EExSwer5cAcAA",
            ]
          ~ column    = 1 -> 0
            height    = 4
            width     = 6
        }
    }
```

This happens when I specify a column value of 0 or leave it blank:

```bash

  column {
    chart_ids = "${signalfx_time_chart.backup_queue_length.*.id}"
    column    = 0
    width     = 6
    height    = 4
  }
}

OR 

```bash

  column {
    chart_ids = "${signalfx_time_chart.backup_queue_length.*.id}"
    width     = 6
    height    = 4
  }
}
```

Explicitly setting it to 1 fixes the issue:

```bash

  column {
    chart_ids = "${signalfx_time_chart.backup_queue_length.*.id}"
    column    = 1
    width     = 6
    height    = 4
  }
}
```

Looking at https://github.com/terraform-providers/terraform-provider-signalfx/blob/master/signalfx/resource_signalfx_dashboard.go#L146, it appears the default in the provider code is **0**. Does the SFX API expect the default value, if unspecified to be **1**?

**Comment by cory-signalfx** [comment]
Eep! Will check it out.

**Comment by cory-signalfx** [comment]
Fixed! This was a bug that's semi-related to #80 :(
